internal/backend/triggers/triggers.go

```diff
@@ -53,3 +97,98 @@ func (m *triggers) Get(ctx context.Context, triggerID sdktypes.TriggerID) (sdkty
 func (m *triggers) List(ctx context.Context, filter sdkservices.ListTriggersFilter) ([]sdktypes.Trigger, error) {
 	return m.db.ListTriggers(ctx, filter)
 }
+
+func (m *triggers) createScheduledWorkflow(ctx context.Context, trigger sdktypes.Trigger, schedule sdktypes.Value) (sdktypes.Trigger, error) {
```

Comment:
I think this code should belong to another module that has access to triggers, not be in triggers themselves.

Reply:
temporal client module?

Reply:
moved